### ISSUE TITLE
Feature Request: Log when manual Authorization is required when using --auth-files

### ISSUE THREAD

**Issue by OscarVanL** [abraunegg/onedrive#2475]
### Is your feature request related to a problem? Please describe.

Sometimes when my onedrive container is restarted it gets stuck at:
```
Initializing the OneDrive API ...
Configuring Global Azure AD Endpoints
Using Curl defaults for all HTTP operations
```
becuse it is waiting for me to re-auth via the `authUrl` and `responseUrl` files.

In this scenario, I'd like to alert myself that I need to re-auth, however when the client restarts and _doesn't_ need to re-auth these logs still get output.

This makes it hard for me to (from machine-parsing logs) differentiate between a restart scenario that needs to be re-authed or a restart scenario that does not require re-authing.

### Describe the solution you'd like

Output something like: `Client requires authentication before proceeding` when the client requires manual re-authing that I can grep for.

### Describe alternatives you've considered

Instead alerting on the absence of the message "Sync with OneDrive is Complete" for a few hours

**Comment by abraunegg** [comment]
@OscarVanL 
The application already prints the following when a `--reauth` is needed, or when the existing tokens are invalid:
```
ERROR: Refresh token invalid, use --reauth to authorize the client again.
```
and
```
Refresh token invalid, use --reauth to authorize the client again
```

It would be worth checking your logs for these messages as they certainly will be printed when the application is advised by the OneDrive API that the existing authentication items are no longer valid - so it should have given you the heads up that the values were invalid.

This being said, there is potentially a gap here that I can see and from what you have pointed out. I will potentially look at this during the the development of v2.5.0 alpha-1 - I am not going to validate any more features and/or additional items with where I am at currently with v2.5.0 alpha-0.

**Comment by abraunegg** [comment]
@OscarVanL 
If you have an opportunity, please can you test v2.5.0-alpha-0.

In this version, when the client is *waiting* for the auth-files to be present, the following will be presented:
```
Client requires authentication before proceeding. Waiting for --auth-files elements to be available.
```

**Comment by abraunegg** [comment]
@OscarVanL
Please can you respond in regards to testing the PR?

**Comment by abraunegg** [comment]
@OscarVanL 

Please can you test the following PR as part of 'onedrive client version v2.5.0-alpha-3' to potentially resolve your issue:

```
git clone https://github.com/abraunegg/onedrive.git
cd onedrive
git fetch origin pull/2520/head:pr2520
git checkout pr2520
```

To run the PR, you need to run the client from the PR build directory:
```
./onedrive <any options needed>
```

When running the PR, your client version should be: `v2.5.0-alpha-3 GitHub version: v2.4.25-35-g3798e1a`

**Comment by OscarVanL** [comment]
Sorry, I can't see a build directory after checking it out. I tried building, but couldn't get it to work.

**Comment by abraunegg** [comment]
Closing this issue and marking as implemented in v2.5.0 alpha series of code

**Comment by abraunegg** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
